kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverReportsConfig.kt

```diff
@@ -650,19 +651,29 @@ public interface KoverReportFilter {
     /**
      * Add to filters all classes and functions marked by specified annotations.
      *
+     * Use cases:
+     *  - in case of exclusion filters all classes or function marked by at least one of the specified annotation will be excluded from the report
```

Comment:
Because now you can exclude class and function by annotation, but you **can't** include function by annotation, only whole class.

Reply:
Why so? Seems unobvious, especially given that `@Exclude` works on functions.

Reply:
There was no use case to think through this behavior.

Inclusion by annotation was added recently and on a specific request.
Perhaps this can be aligned in the future.